.travis.yml

```diff
@@ -1,16 +1,22 @@
 language: go
+sudo: false
 go:
+  - 1.6
+  - 1.7
+  - 1.8
   - tip
-before_install:
-  - go get github.com/mattn/goveralls
-  - go get golang.org/x/tools/cmd/cover
-  - go get github.com/stretchr/testify/assert
-  - go get github.com/Sirupsen/logrus
-  - go get github.com/spf13/viper
-  - go get github.com/tuvistavie/securerandom
-  - go get gopkg.in/mattes/go-expand-tilde.v1
-  - go get github.com/akamai-open/AkamaiOPEN-edgegrid-golang
+matrix:
+  allow_failures:
+    - go: 1.6
+    - go: tip
+addons:
+  apt:
+    sources:
+      - sourceline: 'ppa:masterminds/glide'
+    packages:
+      - glide
 install:
+  - glide install
   - go install .
 script:
   - export PATH="$PATH:$HOME/gopath/bin"
```

Comment:
@njuettner the benefit of using the PPA for the glide install is that we get newer versions without needing to update the hardcoded version you have there (`cd $GOPATH/src/github.com/Masterminds/glide && git checkout v0.12.3 && go install && cd -`)

As for using `goveralls`, I'm all for it, however I think we'll still need the first two lines of the `script` section to setup the `PATH` (though that one might not be necessary) and copy the `sample_edgerc` to `$HOME/.edgerc`.